test/layout.ts

```diff
@@ -556,4 +556,70 @@ describe("Layout Tests", () => {
       callbackNumber++;
     }
   });
+
+  it("checks that images source is not captured if the config disallows it", (done) => {
```

Comment:
Add some test to cover masking of the attributes you added in addition to the src field.